petvax-services/src/main/java/com/petvax/petvaxServices/converter/PetsConverter.java

```diff
@@ -1,21 +1,43 @@
 package com.petvax.petvaxServices.converter;
 
+import com.petvax.petvaxServices.dto.BreedsResponse;
 import com.petvax.petvaxServices.dto.PetsResponse;
+import com.petvax.petvaxServices.dto.SpeciesResponse;
+import com.petvax.petvaxServices.dto.VaccinationsResponse;
+import com.petvax.petvaxServices.entity.BreedEntity;
 import com.petvax.petvaxServices.entity.PetEntity;
+import com.petvax.petvaxServices.entity.SpeciesEntity;
+import com.petvax.petvaxServices.entity.VaccinationEntity;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class PetsConverter {
 
+    @Autowired
+    private SpeciesConverter speciesConverter;
+
+    @Autowired
+    private BreedsConverter breedsConverter;
+
     public PetsResponse convertPetsToPetResponse(final PetEntity petEntity) {
 
         // Creates response with returned value based upon request
         PetsResponse.Builder petResponseBuilder = new PetsResponse.Builder();
 
+        // Adds species details to response using the species response
+        SpeciesEntity species = petEntity.getSpeciesEntity();
```

Comment:
Can we remove SpeciesEntity since it's already included in the breedsResponse? Also, can we reorder the fields in the response so the output results in this?

```
{
    "uuid": "a90a655e-b25b-11ed-8531-0242ac120002",
    "name": "Huck",
    "ownerID": "f1e18004-dc0b-443f-89e7-dc0c16734518",
    "breed": {
        "uuid": "c523cc7e-a5ed-4279-bda1-065c224abae4",
        "name": "Labrador Retriever",
        "species": {
            "uuid": "46e45c98-758f-4ff9-9811-389e5b206e0c",
            "speciesName": "Canine"
        }
    },
    "age": "17",
    "active": true
}
```